RandomUsersTest/app/src/androidTest/java/com/skaravatskiy/randomusers/RoomTestNegative.kt

```diff
@@ -0,0 +1,51 @@
+package com.skaravatskiy.randomusers
+
+import android.arch.persistence.room.Room
+import android.support.test.InstrumentationRegistry
+import android.support.test.runner.AndroidJUnit4
+import com.google.gson.Gson
+import com.skaravatskiy.randomusers.data.model.common.Mapper
+import com.skaravatskiy.randomusers.data.model.database.AppDatabase
+import com.skaravatskiy.randomusers.data.model.database.User
+import com.skaravatskiy.randomusers.data.model.database.UsersDao
+import com.skaravatskiy.randomusers.data.model.poko.UsersResponse
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import kotlin.random.Random
+
+@RunWith(AndroidJUnit4::class)
+class RoomTestNegative {
```

Comment:
We no need to create separate files for positive and negative test cases, please use one file